src/Reducers/requestAssetsReducers.js

```diff
@@ -62,7 +62,27 @@ const requestAssetsReducers = (state = initialState.requestAssets, action) => {
     case 'HANDLE_SELECTED_EMPLOYEES': {
       return { ...state, selectedEmployees: action.payload };
     }
-
+    case 'HANDLE_MODAL_OPEN': {
+      return { ...state, openModal: !state.openModal };
+    }
+    case 'HANDLE_MODAL_CLOSE': {
+      const searchResultCopy = state.selectedSearchResultCopy.reduce(
+        (acc, value) => {
+          if (!value.chec) acc.push(value);
+          return acc;
+        },
+        []
+      );
+      return {
+        ...state,
+        selectedSearchResultCopy: searchResultCopy,
+        openModal: !state.openModal,
+        notification: !state.notification
+      };
+    }
```

Comment:
To my understanding, `handleModalClose` seems to be just removing those items from the requestsAssets table. Maybe `HANDLE_REMOVE_SELECTED`? I think `HANDLE_MODAL_OPEN` makes more sense as `HANDLE_MODAL_TOGGLE`. 

I suggest these only because reading through the rest of the code I was having a hard time understanding the container bindings.

Reply:
I see, I will be changing the names what you suggested it.